Objective-C.gitignore

```diff
@@ -1,62 +0,0 @@
-# Xcode
-#
-# gitignore contributors: remember to update Global/Xcode.gitignore, Objective-C.gitignore & Swift.gitignore
-
-## Build generated
-build/
-DerivedData/
-
-## Various settings
-*.pbxuser
-!default.pbxuser
-*.mode1v3
-!default.mode1v3
-*.mode2v3
-!default.mode2v3
-*.perspectivev3
-!default.perspectivev3
-xcuserdata/
-
-## Other
-*.moved-aside
-*.xcuserstate
-
-## Obj-C/Swift specific
-*.hmap
-*.ipa
-*.dSYM.zip
-*.dSYM
-
-# CocoaPods
```

Comment:
Sorry, I should have been clearer here. There's rules that exit both in `Swift.gitignore` and `Objective-C.gitignore` that will disappear because they aren't in `Global/XCode.gitignore` - we need to ensure these files are all in sync before we can switch over to symlinks. Make sense?